docs/changehistory/NextVersion.md

```diff
@@ -76,6 +77,13 @@ Reduced API surface of an `ElectronApp` class to only allow white-listed APIs fr
 
 The argument for [IModelHost.startup]($backend) has been changed from [IModelHostConfiguration]($backend) to the [IModelHostOptions]($backend) interface. This matches the approach on the frontend for [IModelApp.startup]($frontend) and makes it easier to supply startup options. `IModelHostConfiguration` implements `IModelHostOptions`, so existing code will continue to work without changes.
 
+## Cloud storage changes
+The existing beta implementation of cloud storage tile cache ([CloudStorageService]($backend) - [AzureBlobStorage]($backend), [AliCloudStorageService]($backend)) has been deprecated in favor of [iTwin/object-storage](https://github.com/iTwin/object-storage).
+
+`IModelHost.tileCacheService` remains to support older frontends, in which case `IModelHost.tileStorage` still has to be setup. Both are taken care of if `IModelHostOptions.tileCacheAzureCredentials` are used.
```

Comment:
```suggestion
[IModelHost.tileCacheService]($backend) remains to support older frontends, in which case [IModelHost.tileStorage]($backend) still has to be setup. Both are taken care of if [IModelHostOptions.tileCacheAzureCredentials]($backend) is used to initialize the tile cache.
```

Reply:
Please explain at a high-level the motivation for / benefits of this change (e.g., cloud-agnostic).